tests/phpunit/includes/bootstrap.php

```diff
@@ -15,6 +15,10 @@ function() use ( $_root_dir ) {
 define( 'WP_TESTS_PHPUNIT_POLYFILLS_PATH', $_root_dir . '/vendor/yoast/phpunit-polyfills/' );
 require_once $_tests_dir . '/includes/bootstrap.php';
 
+if ( ! defined( 'PLL_CACHE_HOME_URL' ) ) {
+	define( 'PLL_CACHE_HOME_URL', false );
+}
```

Comment:
Indeed, it can also be added in `phpunit.xml` with some special tags.£
But for the moment I'm not sure it should be left and merged. See https://github.com/polylang/polylang-pro/issues/1703

Reply:
But without this it can't be tested can it?

Reply:
Nope, leaving it depends whether or not we'll address https://github.com/polylang/polylang-pro/issues/1703 shortly.
@Chouby Any thought?